dpnp/tests/test_histogram.py

```diff
@@ -1109,5 +1108,5 @@ def test_different_bins_amount(self, bins_count):
             ix, iy, bins=bins_count
         )
         assert_array_equal(result_hist, expected_hist)
-        assert_allclose(result_edges_x, expected_edges_x, rtol=1e-6)
-        assert_allclose(result_edges_y, expected_edges_y, rtol=1e-6)
+        assert_allclose(result_edges_x, expected_edges_x)
```

Comment:
It seems failing on Iris Xe ([log](https://cje-fm-owrp-prod04.devtools.intel.com/satg-dap-intelpython/job/intel-packages/job/dpnp/job/dev-linux-py3.11/job/test-wheel-conda-stable/255)), while both result array have `float32` dtype.
It might be because there is comparison against latest numpy 2.2.3 in the impacted use case.

The same issue is visible on Windows ([log](https://cje-fm-owrp-prod04.devtools.intel.com/satg-dap-intelpython/job/intel-packages/job/dpnp/job/dev-windows-py3.11/job/test-wheel-conda-stable/226/cloudbees-pipeline-explorer/)) with Iris Xe.